locales/zh_CN/LC_MESSAGES/meta.po

```diff
@@ -149,41 +176,72 @@ msgid ""
 " x[f + 1, p] = x[f, p] + dt * v[f + 1, p]\n"
 " C[f + 1, p] = new_C"
 msgstr ""
+"@ti.kernel\n"
+"def g2p(f: ti.i32):\n"
+"for p in range(0, n_particles):\n"
+" base = ti.cast(x[f, p] * inv_dx - 0.5, ti.i32)\n"
+" fx = x[f, p] * inv_dx - ti.cast(base, real)\n"
+" w = [0.5 * ti.sqr(1.5 - fx), 0.75 - ti.sqr(fx - 1.0),\n"
+"      0.5 * ti.sqr(fx - 0.5)]\n"
+" new_v = ti.Vector([0.0, 0.0])\n"
+" new_C = ti.Matrix([[0.0, 0.0], [0.0, 0.0]])\n"
+"\n"
+" # 展开了9次迭代来获得更高的性能\n"
+" for i in ti.static(range(3)):\n"
+"   for j in ti.static(range(3)):\n"
+"     dpos = ti.cast(ti.Vector([i, j]), real) - fx\n"
+"     g_v = grid_v_out[base(0) + i, base(1) + j]\n"
+"     weight = w[i](0) * w[j](1)\n"
+"     new_v += weight * g_v\n"
+"     new_C += 4 * weight * ti.outer_product(g_v, dpos) * inv_dx\n"
+"\n"
+" v[f + 1, p] = new_v\n"
+" x[f + 1, p] = x[f, p] + dt * v[f + 1, p]\n"
+" C[f + 1, p] = new_C"
 
 #: ../../meta.rst:95
 msgid "When to use for loops with ``ti.static``"
-msgstr ""
+msgstr "何时使用 ``ti.static`` 来进行for循环"
 
 #: ../../meta.rst:97
 msgid "There are several reasons why ``ti.static`` for loops should be used."
-msgstr ""
+msgstr "下面有一些为何应该在for循环的同时使用``ti.static``的原因。"
 
 #: ../../meta.rst:99
 msgid "Loop unrolling for performance."
-msgstr ""
+msgstr "循环展开以提高性能。"
 
 #: ../../meta.rst:100
 msgid ""
 "Loop over vector/matrix elements. Indices into Taichi matrices must be a "
 "compile-time constant. Indexing into taichi tensors can be run-time "
-"variables. For example, if ``x`` is a 1-D tensor of 3D vector, accessed "
-"as ``x[tensor_index][matrix index]``. The first index can be variable, "
-"yet the second must be a constant."
+"variables. For example, if ``x`` is a 1-D tensor of 3D vector, accessed as "
+"``x[tensor_index][matrix index]``. The first index can be variable, yet "
+"the second must be a constant."
 msgstr ""
+"对向量/矩阵的元素进行循环。对Taichi矩阵上的索引必须为编译时常量。对Taichi张"
+"量上的索引可以为运行时变量。例如，如果 ``x`` 是一个3维向量的1维张量，并可被"
+"以 ``x[tensor_index][matrix_index]`` 的形式访问。第一个索引（tensor_index）"
+"可以是变量，但是第二个索引（matrix_index）必须是一个常量。"
 
 #: ../../meta.rst:102
 msgid "For example, code for resetting this tensor of vectors should be"
-msgstr ""
+msgstr "例如，重置向量的张量（tensor of vectors）的代码应该为"
```

Comment:
TBH I thought this was `(重置向量)的张量` instead of `重置(向量的张量)` at the first look... Let's move the position of `的` to make this clear.
```suggestion
msgstr "例如，向量张量（tensor of vectors）的重置代码应该为"
```
It would be good to consider `向量张量` as a term, wdyt?
Btw, I thought we translate `vector` -> `矢量`? Not sure if we want to clarify this in wiki.

Reply:
Thanks!  `向量张量` as a single term makes it much more clear now.

Searching across the existing docs I haven't found 矢量 yet. But I think either 向量 or 矢量 works, we just need to standardize it to have a unified style given how often this word appears in the original docs.